Zend/zend_vm_def.h

```diff
@@ -3894,8 +3908,12 @@ ZEND_VM_HOT_HANDLER(129, ZEND_DO_ICALL, ANY, ANY, SPEC(RETVAL))
 	}
 #endif
 
+ZEND_VM_C_LABEL(do_icall_cleanup):
 	EG(current_execute_data) = execute_data;
 	zend_vm_stack_free_args(call);
+	if (UNEXPECTED(ZEND_CALL_INFO(call) & ZEND_CALL_HAS_EXTRA_NAMED_PARAMS)) {
```

Comment:
The second additional checks for each ICALL looks terrible.
Can we do this as a part ZPP API of internal function with variadic arguments?